sys/linux/auto.txt

```diff
@@ -1512,7 +1513,7 @@ bpf_attr_3$auto_record {
 	kern_version		int32
 	prog_flags		int32
 	prog_name		array[int8, 16]
-	prog_ifindex		ifindex
+	prog_ifindex		int32
```

Comment:
Was the `ifindex` -> `int32` change intended? Here and in other places.

Reply:
No. I'll fix that.

Reply:
It should be fixed now, fd should be handled too. No extra code was required.